deployment/migration-assistant-solution/lib/solutions-stack.ts

```diff
@@ -162,9 +176,33 @@ export class SolutionsInfrastructureStack extends Stack {
             role: bootstrapRole
         })
 
-        let vpc;
+        let vpc: IVpc;
         if (props.createVPC) {
-            vpc = new Vpc(this, 'Vpc', {});
+            vpc = new Vpc(this, 'Vpc', {
+                ipProtocol: IpProtocol.DUAL_STACK
+            });
+            // S3 used for storage and retrieval of snapshot data for backfills
+            new GatewayVpcEndpoint(this, 'S3VpcEndpoint', {
+                service: GatewayVpcEndpointAwsService.S3,
+                vpc: vpc,
+            });
+
+            const serviceEndpoints = [
```

Comment:
Only added endpoints with substantive amount of traffic.  We had many more in the MA network-stack, but these are really the only ones that would see a price difference.  Its a 1/3 the cost to use the VPCe compared to the NAT Gateway.